credits.txt

```diff
@@ -61,3 +66,117 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that makes
+so much of this work possible.
+
+Proof Readers & Editors
+  TheMarksman
+
+Mission & Story Creators
```

Comment:
Becca for the Ka'het and a boarding mission. (#4716, #4735)
Arachi and Jana for Nanachi. (#4685)
Pointedstick for various jobs and smaller missions. (#1777, #3235, #3347, #3359)
Unfettered One for various spaceport mission. (#3151, had a hand in #3298, #3781)
MessyMix for an additional mission during the Wanderer campaign. (#2427)
Petervdmeer for the Hauler IV missions. (#4446)
Fzzr for a mission where you capture a smuggler. (#3475)
Anarchist2 for a Wanderer mission. (#4762)
Czartrak for some pirate jobs. (#3984)

Reply:
Thanks, I've added all those in.